app/bundles/PointBundle/Form/Type/PointType.php

```diff
@@ -33,17 +39,20 @@ class PointType extends AbstractType
     private $security;
 
     /**
-     * @var \Symfony\Bundle\FrameworkBundle\Translation\Translator
+     * @var \Symfony\Component\Translation\TranslatorInterface
      */
     private $translator;
 
     /**
-     * @param MauticFactory $factory
+     * PointType constructor.
+     *
+     * @param \Symfony\Component\Translation\TranslatorInterface      $translator
+     * @param \Mautic\CoreBundle\Security\Permissions\CorePermissions $security
```

Comment:
```suggestion
     * @param CorePermissions $security
```
Use relative path as the classes are defined in the use statements above.